docs/sink_config_options.rst

```diff
@@ -3,6 +3,19 @@
 JDBC Sink Configuration Options
 -------------------------------
 
+Security
+^^^^^^^^
+
+In the connector configuration you will notice there are no security parameters. This is because
+SSL is not part of the JDBC standard and will depend on the JDBC driver in use. In general, you
+will need to configure SSL via the ``connection.url`` parameter. For example, with mysql it would
```

Comment:
nit: mysql should either be monotype font to indicate protocol in the connection url or MySQL to match the product name